packages/react-sdk/src/css/normalize-type-check.ts

```diff
@@ -7,7 +7,8 @@ import type { Style } from "@webstudio-is/css-engine";
 const normalizeWithKeyof = { ...normalize };
 
 type ExportedTags = keyof typeof normalizeWithKeyof;
+type ExpectedTags = HtmlTags | "checkbox" | "radio";
```

Comment:
To not break this semantic you can just inline those presets into components.

Reply:
Sure the question is whats the rightful place for this stuff? what is this typecheck even doing?

Type checking this is hard because we only use tags and those are the same tag with different type combination, to get a type checker here we need {tag, type} object

Reply:
discussed, we are keeping those as exceptions, added a comment to explain